prediction/VehiclesCollisionEvent[] collision_event_list
